.github/workflows/release.yml

```diff
@@ -26,9 +26,9 @@ jobs:
         run: git fetch --prune --unshallow
       -
         name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
-          go-version: 1.14
+          go-version: 1.16
```

Comment:
Probably no reason to stay on go 1.16 - it's way out of date. Should go to 1.20 as it is the latest supported version. Since plugins are just external binaries to Terraform, there is no need to keep it older for compatibility reasons or anything.